npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
Log files should not be checked in. Please remove it and add it to `.gitignore`.